templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -131,6 +134,10 @@ Parameters:
       can include numbers, lowercase letters, uppercase letters, and hyphens (-).
       It cannot start or end with a hyphen (-).
     Type: String
+  QSS3BucketRegion:
+    Default: ''
+    Type: String
+    Description: The AWS Region where the Quick Start S3 bucket is hosted.
```

Comment:
Because of the need to maintain backward compatibility we cannot use a default or a nice drop down of values.